src/main/java/fi/hsl/transitlog/hfp/Main.java

```diff
@@ -37,4 +42,23 @@ public static void main(String[] args) {
         }
 
     }
+
+    private static String readConnectionString() throws Exception {
```

Comment:
Perhaps we should've put this function to commons... ConfigUtils.readFromDockerSecrets or something

Reply:
I agree, after all it is used in seven apps...